constellations/gemini-admin/src/utils/utils.js

```diff
@@ -0,0 +1,19 @@
+export const generatePassword = () => {
```

Comment:
Let's rename this file userUtils.js. Maybe other utility functions can be placed in a different util file.